src/IRTS/CodegenC.hs

```diff
@@ -83,7 +83,7 @@ codegenC' defs out exec incs objs libs flags exports iface dbg
              let args = gccDbg dbg ++
                         gccFlags iface ++
                         -- # Any flags defined here which alter the RTS API must also be added to config.mk
-                        ["-DHAS_PTHREAD", "-DIDRIS_ENABLE_STATS",
+                        ["-std=c99", "-D_POSIX_C_SOURCE=200809L", "-DHAS_PTHREAD", "-DIDRIS_ENABLE_STATS",
```

Comment:
What is the `"-D_POSIX_C_SOURCE=200809L"` flag?

Reply:
The multi thread code uses some very recent POSIX functions such as `pthread_cond_timedwait` and `pthread_mutexattr_settype` that are only available after this feature is selected.